source/2019-11-22-the-ember-times-issue-125.md

```diff
@@ -58,13 +59,11 @@ You can read the full interview on [the Ember Discuss Forum](update me!).
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Beta Releases for Ember Basic Dropdown and Ember Power Select ⭐](https://twitter.com/MiguelCamba/status/1196520042948628480)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Miguel Camba @cibernox](https://github.com/cibernox) released beta versions of [ember-basic-dropdown](https://github.com/cibernox/ember-basic-dropdown) (3.0.0-beta.1) and [ember-power-select](https://github.com/cibernox/ember-power-select) (4.0.0-beta.1) which are now glimmer components!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Both beta versions require [Ember 3.13](https://blog.emberjs.com/2019/09/25/ember-3-13-released.html) or greater. One caveat is that the beta version of Ember Power Select may throw an error that `assign` is not a helper. If you encounter this, installing [ember-assign-helper](https://github.com/cibernox/ember-assign-helper) should help.
```

Comment:
@chrisrng Miguel just released 4.0.0-beta.3 version, which fixes this error. Can we point the readers to use the .3 version above? Thanks.

Reply:
haha damn already out of date - resolved!